vcbuild.bat

```diff
@@ -14,7 +14,7 @@ if /i "%1"=="/?" goto help
 @rem Process arguments.
 set config=Release
 set target=Build
-set target_arch=x86
+set target_arch=x64
```

Comment:
The `configure` script checks some environment variables on Windows: https://github.com/nodejs/node/blob/master/configure#L735-L748